### ISSUE TITLE
TypeScript Definition Error: Client lacks call or construct signature

### ISSUE THREAD

**Issue by shellscape** [sendgrid/sendgrid-nodejs#1086]
### Issue Summary

The TypeScript definitions for the project don't allow instantiating a new `Client` according to `tsc`. This is fine however, in vanilla Node.

### Steps to Reproduce

For demonstration purposes I'll be using the `node` and `ts-node` REPLs

Successful in Node:

```console
→ node
Welcome to Node.js v12.15.0.
Type ".help" for more information.
> const client  = require('@sendgrid/client')
undefined
> client
Client {
  apiKey: '',
  defaultHeaders: { Accept: 'application/json', 'User-agent': 'sendgrid/6.4.0;nodejs' },
  defaultRequest: {
    json: true,
    baseUrl: 'https://api.sendgrid.com/',
    url: '',
    method: 'GET',
    headers: {}
  },
  Client: [Function: Client]
}
> new client.Client()
Client {
  apiKey: '',
  defaultHeaders: { Accept: 'application/json', 'User-agent': 'sendgrid/6.4.0;nodejs' },
  defaultRequest: {
    json: true,
    baseUrl: 'https://api.sendgrid.com/',
    url: '',
    method: 'GET',
    headers: {}
  }
}
```

But fails within TypeScript:

```console
→ ts-node
> import client from '@sendgrid/client'
{}
> client
Client {
  apiKey: '',
  defaultHeaders: { Accept: 'application/json', 'User-agent': 'sendgrid/6.4.0;nodejs' },
  defaultRequest: {
    json: true,
    baseUrl: 'https://api.sendgrid.com/',
    url: '',
    method: 'GET',
    headers: {}
  },
  Client: [Function: Client]
}
> client.Client
[Function: Client]
> new client.Client
[eval].ts:4:1 - error TS2351: Cannot use 'new' with an expression whose type lacks a call or construct signature.

4 new client.Client
  ~~~~~~~~~~~~~~~~~

undefined
> 
```

_Note: This example requires that you have a local tsconfig that has `esModuleInterop: true` or are using the ts-node flag of the same name. Otherwise, you have to use `import * as client from...`_

### Code Snippet

see above

### Exception/Log

see above

### Technical details:
* sendgrid-nodejs version: @sendgrid/client v6.4.0
* node version: v12.15.0

**Comment by childish-sambino** [comment]
Fixed by https://github.com/sendgrid/sendgrid-nodejs/pull/1040 released in [6.5.3](https://github.com/sendgrid/sendgrid-nodejs/releases/tag/6.5.3)

**Comment by shellscape** [comment]
Sorry about that, I could have swore we had the latest installed.
